lua/terrortown/entities/roles/ttt_role_base/init.lua

```diff
@@ -60,3 +60,13 @@ function ROLE:GetAvailableRoleCount(ply_count)
 	-- make sure there is at least 1 of the role
 	return math.Clamp(role_count, 1, maxAmount)
 end
+
+---
+-- Use this hook to make a role nonselectable.
```

Comment:
```suggestion
-- Use this hook to make a role unselectable.
```

Reply:
what is your reason for this suggestion? both are valid imo (ful disclosure: i made the `nonselectable` suggestion =D )